core/http-auth-spi/src/main/java/software/amazon/awssdk/http/auth/spi/HttpSigner.java

```diff
@@ -31,6 +32,12 @@
 @SdkPublicApi
 public interface HttpSigner<IdentityT extends Identity> {
 
+    /**
+     * A {@link Clock} to be used at the time of signing which is shared among several signers. This property defaults to the time
```

Comment:
nit: The "which is shared among several signers" can read like the Clock is shared as opposed to the property being used by multiple signers. I think avoiding "shared" here is better.

Suggest something like:
```
* A {@link Clock} to be used to derive the signing time. This property defaults to the system clock.
*
*<p>Note, signing time may not be relevant to some signers.
```

Reply:
I just pushed a commit.